### DISCUSSION TITLE
Quarto bullets with parentheses has unexpected behavior

### DISCUSSION THREAD

**Discussion by shannonpileggi** [quarto-dev/quarto-cli#10406] [Q&A] [answered]
### Description

I was making some bullets with parentheses adjacent to the asterisk and I observed some unexpected behavior such that the parentheses didn't print and the indenting on the bullets adjusted. 

Is this a function of some markdown type syntax I am not aware of? Perhaps capabilities to make custom bullets/indentation? Is there a way to leverage what is contained in the parentheses as a "bullet" but omit printing of the closed circle?

![image](https://github.com/user-attachments/assets/bc4a84f0-5f44-4046-a618-003a18ed9b18)

````qmd
---
title: "Untitled"
format: html
editor: visual
---

## Resources


* (2024) resource from 2024

* (2022) resource from 2022

* regular resource

````

I am using quarto version 1.3.450 on Windows 11.

**Comment by mcanouil** [comment, marked as answer]
I believe you nested two lists: a list with asterisks and one with numeric (startnum).
That's Pandoc markdown extended syntax.

https://pandoc.org/MANUAL.html#lists

You can check that by inspecting the HTML code produced.

**Reply by shannonpileggi** [reply]
Got it, thank you very much!

**Comment by Steinthal** [comment]
I agree with **mcanouil**, see the test entry below and the dots behind your numbers. 
You need to escape at least the closing parentheses:

````
---
title: "Untitled"
format: html
---

## Resources

* (2024\) resource from 2024

* (2022\) resource from 2022

* regular resource

* 202) Test
````

![Example](https://github.com/user-attachments/assets/c85af63f-1a94-4ab0-8b24-44db55fc9a13)

**Reply by shannonpileggi** [reply]
Thank you!
